app/src/main/java/org/mozilla/focus/widget/BrowserNavigationOverlay.kt

```diff
@@ -86,6 +86,11 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
             if (it == null) { return@forEach }
             it.setTint(ContextCompat.getColor(context, R.color.button_default))
         }
+
+        navUrlInput.compoundDrawablesRelative.forEach {
+            if (it == null) { return@forEach }
+            it.setTint(ContextCompat.getColor(context, R.color.button_text_color))
```

Comment:
nit: `it?.setTint(...` and rm null check, here and above.

fwiw, I'm not sure if return will return from the current function call (likely) or the entire `forEach` iteration.